CardsCashCasino/Util/DisplayIndicatorUtil.cs

```diff
@@ -122,6 +132,7 @@ public static void LoadContent(ContentManager content)
             EightTexture = content.Load<Texture2D>("eight");
             NineTexture = content.Load<Texture2D>("nine");
             DollarSignTexture = content.Load<Texture2D>("dollarSign");
+            TurnTexture = content.Load<Texture2D>("TEMP_CHIP");
```

Comment:
since we are already loading it here, why are we loading it in the constructor above? I left a comment about this already.